html/recordplaytest.js

```diff
@@ -497,6 +497,18 @@ function startRecording() {
 					recordplay.hangup();
 				}
 			});
+
+			$('#pause').on('click', () => {
+				recordplay.send({message: {request: 'pause'}});
+				$('#pause').attr('hidden', true);
+				$('#resume').attr('hidden', false);
+			});
+
+			$('#resume').on('click', () => {
+				recordplay.send({message: {request: 'resume'}});
+				$('#pause').attr('hidden', false);
+				$('#resume').attr('hidden', true);
+			});
```

Comment:
No need for the empty lines above. As a side note, the way you implemented this won't work for consecutive recordings: new click handlers will be added any time you start one, which means that any time you click the button it will send the same request multiple times. The way we handle this in other demos is via something like:

    .unbind('click').click(function() {

As a side note, I'd prefer the pause/resume to be a single one with a toggle (e.g., like the audio/video on/off buttons in the EchoTest demo, and others), but that's not important.

Reply:
PS: we use the bootstrap `hide` class to hide/show elements. Please use that instead of the manual `hidden` attribute.